CreeDictionary/CreeDictionary/templates/CreeDictionary/components/header.html

```diff
@@ -3,13 +3,13 @@
 <div class="top-bar app__header">
   <header class="branding top-bar__logo"><a href="{% url 'cree-dictionary-index' %}">
       <h1 class="branding__heading branding__title">{% orth 'itwêwina' %}</h1>
-      <p class="branding__heading branding__subtitle">Plains Cree Dictionary</p>
+      <p class="branding__heading branding__subtitle" role="doc-subtitle">Plains Cree Dictionary</p>
   </a></header>
-  {# TODO: The language and orthography selectors are not ready yet, so hide them: #}
+
   <nav class="top-bar__nav">
     <details class="toggle-box toggle-box--with-menu">
       <summary id="language-selector__button" class="toggle-box__toggle" data-cy="language-selector"
```

Comment:
`<summary>` tags inherintly have the [ARIA role=button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary#:~:text=Implicit%20ARIA%20role)!